topo_processor/metadata/metadata_loaders/metadata_loader_imagery_historic.py

```diff
@@ -109,6 +106,31 @@ def read_csv(self, metadata_file: str = "") -> None:
 
         self.is_init = True
 
+    def add_centroid(self, item: Item, asset_metadata: Dict[str, str]):
+        centroid = {
+            "lat": string_to_number(asset_metadata.get("photocentre_lat", None)),
+            "lon": string_to_number(asset_metadata.get("photocentre_lon", None)),
+        }
+        if not all(centroid.values()) or "" in centroid.values():
```

Comment:
would be good to validate that these are actual lat/lon and not inverted lon/lat or random numbers?

also is `0` considered a warning?

Reply:
I added some checks to validate the numbers are within either -90-90 or -180-180. We could make this more specific to NZ (but as is crosses the dateline the nz bbox is not quite accurate)?